integration_tests/pages/page.ts

```diff
@@ -46,4 +46,15 @@ export default abstract class Page {
   getMessages = (): PageElement => cy.get('.moj-alert')
 
   mojAlertBody = (): PageElement => cy.get('.moj-alert__content')
+
+  protected clickDisabledOnSubmitButton = (dataTestName: string): void => {
```

Comment:
Moving this the `Page` class so checking a button is disabled on submit code can be reused across other pages.